.github/actions/parse-distrib/action.yml

```diff
@@ -32,43 +35,50 @@ runs:
           PACKAGE_VERSION_SEPARATOR="-"
           PACKAGE_DISTRIB_NAME="el7"
           PACKAGE_EXTENSION="rpm"
-          DISTRIB_FAMILY="el"
+          DISTRIB_FAMILY="el7"
+          DISTRIB_NAME="centos7"
```

Comment:
this condition could be removed later on, as it does not seems to be any EL7 related package remaining